modules/nw-multus-configuring-whereabouts-ip-reconciler-schedule.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+= Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that might result in exhausting IPs and therefore prevent new Pods from getting an IP allocated to them.
+
+Use this procedure to change the frequency at which the IP reconciler runs. 
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have deployed the `whereabouts-reconciler` daemon set, and the `whereabouts-reconciler` pods are up and running.
```

Comment:
(Now that I've seen the other procedure) Seems like this could match the style of "the Whereabouts reconciler daemon set" used in the other procedure.
```suggestion
* You have deployed the Whereabouts reconciler daemon set, and the `whereabouts-reconciler` pods are up and running.
```

Reply:
looks like it should `whereabouts-reconciler`  when talking about the daemon set